src/main/java/com/stolensugar/web/controller/Controller.java

```diff
@@ -124,4 +111,17 @@ public ResponseEntity<?> createSpokenForm(@Valid @RequestBody List<SpokenFormMod
 
         return new ResponseEntity<>(spokenFormActivity.execute(spokenFormRequest), HttpStatus.OK);
     }
+
+    @GetMapping(value = "/flashcards/{id}", produces = {"application/json"})
+    public ResponseEntity<?> GetMochiDeck(@PathVariable String id, @RequestParam(value = "app", required = false) String app) throws Exception {
```

Comment:
I don't think we should throw an Exception here. If we do, where is this handled?